python/langsmith/run_trees.py

```diff
@@ -173,6 +173,7 @@ def set(
         outputs: Optional[Mapping[str, Any]] = NOT_PROVIDED,
         tags: Optional[Sequence[str]] = NOT_PROVIDED,
         metadata: Optional[Mapping[str, Any]] = NOT_PROVIDED,
+        usage: Optional[ls_schemas.ExtractedUsageMetadata] = NOT_PROVIDED,
```

Comment:
thoughts on calling it usage_metadata? to match the actual metadata key

Reply:
Yeah probably a better idea - was hoping we could transition to just `usage` in the future but probably better to do it all at once